scripts/azure_new_vm_size_provision/watcher-new-azure-vm-size.ps1

```diff
@@ -0,0 +1,84 @@
+﻿if ( $customSecretsFilePath )
+{
+	$secretsFile = $customSecretsFilePath
+	Write-Host "Using provided secrets file: $($secretsFile | Split-Path -Leaf)"
+}
+if  ($env:Azure_Secrets_File)
+{
+	$secretsFile = $env:Azure_Secrets_File
+	Write-Host "Using predefined secrets file: $($secretsFile | Split-Path -Leaf) in Jenkins Global Environments."
+}
+if ( $secretsFile -eq $null )
+{
+    Write-Host "ERROR: Azure Secrets file not found in Jenkins / user not provided -customSecretsFilePath" -ForegroundColor Red -BackgroundColor Black
+    exit 1
+}
+
+
+if ( Test-Path $secretsFile)
+{
+	Write-Host "$($secretsFile | Split-Path -Leaf) found."
+    $xmlSecrets = [xml](Get-Content $secretsFile)
+    .\AddAzureRmAccountFromSecretsFile.ps1 -customSecretsFilePath $secretsFile
```

Comment:
are you sure you will be in the same folder with this script?
Instead of using relative paths, you can add a parameter to specify the path to the LISAv2 folder, and execute the script using that path.

Reply:
Fixed